client/app/utils/CheckSignedInContainer.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import toastr from 'toastr';
+import { createBrowserHistory } from 'history';
+
+const history = createBrowserHistory();
+
+export default (ComposedComponent) => {
+  class CheckSignedInContainer extends React.Component {
+    componentWillMount() {
```

Comment:
In most situations, componentDidMount is a preferred choice. Do you have special reason behind your use of componentWillMount here?